src/Shouldly.Tests/Strings/ShouldNotMatch.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.Strings
+{
+    public class ShouldNotMatch : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldThrowAWobbly()
+        {
+            "Cheese".ShouldNotMatch(@"\w+", "Some additional context");
+        }
+
+        protected override string ChuckedAWobblyErrorMessage
+        {
+            get
+            {
+                return "\"Cheese\" should not match \"\\w+\" but was \"Cheese\" " +
```

Comment:
I'm sorry. I've gotten so busy with school work. I think it would be best if you finished it.